components/PriceRanges/PriceRangesForm.js

```diff
@@ -0,0 +1,59 @@
+var React = require('react');
+
+class PriceRangesForm extends React.Component {
+  getInput(type) {
+    return (
+      <label className={this.props.cssClasses.label}>
+        <span className={this.props.cssClasses.currency}>{this.props.labels.currency} </span>
+        <input className={this.props.cssClasses.input} ref={type} type="number" />
+      </label>
+    );
+  }
+
+  handleSubmit(event) {
+    let from = +this.refs.from.value || undefined;
+    let to = +this.refs.to.value || undefined;
+    this.props.refine(from, to, event);
+  }
+
+  render() {
+    let fromInput = this.getInput('from');
+    let toInput = this.getInput('to');
+    let onSubmit = this.handleSubmit.bind(this);
+    return (
+      <form className={this.props.cssClasses.form} onSubmit={onSubmit} ref="form">
+        {fromInput}
+        <span className={this.props.cssClasses.separator}> {this.props.labels.separator} </span>
+        {toInput}
+        <button className={this.props.cssClasses.button} type="submit">{this.props.labels.button}</button>
```

Comment:
I'm happy with that; it's the same for the checkboxes in the refinementList widget anyway.